### ISSUE TITLE
No CRD displayed in k8s 1.22.2

### ISSUE THREAD

**Issue by foal** [harbur/kubernetic#327]
I have deployed the cert-manager but do not see any custom resources defined by it.

Kubernetic: 3.1.0.0
kubectl version
```
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.2", GitCommit:"8b5a19147530eaac9476b0ab82980b4088bbc1b2", GitTreeState:"clean", BuildDate:"2021-09-15T21:38:50Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.2", GitCommit:"8b5a19147530eaac9476b0ab82980b4088bbc1b2", GitTreeState:"clean", BuildDate:"2021-09-15T21:32:41Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}
```
C:\Temp>kubectl get CustomResourceDefinition
```
NAME                                                  CREATED AT
apiservers.operator.tigera.io                         2021-10-02T15:34:36Z
apprepositories.kubeapps.com                          2021-10-02T16:07:35Z
bgpconfigurations.crd.projectcalico.org               2021-10-02T15:34:35Z
bgppeers.crd.projectcalico.org                        2021-10-02T15:34:35Z
blockaffinities.crd.projectcalico.org                 2021-10-02T15:34:35Z
catalogsources.operators.coreos.com                   2021-10-03T08:31:57Z
certificaterequests.cert-manager.io                   2021-10-03T10:54:00Z
certificates.cert-manager.io                          2021-10-03T10:54:00Z
challenges.acme.cert-manager.io                       2021-10-03T10:54:00Z
clusterinformations.crd.projectcalico.org             2021-10-02T15:34:35Z
clusterissuers.cert-manager.io                        2021-10-03T10:54:00Z
clusterserviceversions.operators.coreos.com           2021-10-03T08:31:57Z
felixconfigurations.crd.projectcalico.org             2021-10-02T15:34:35Z
globalnetworkpolicies.crd.projectcalico.org           2021-10-02T15:34:35Z
globalnetworksets.crd.projectcalico.org               2021-10-02T15:34:35Z
hostendpoints.crd.projectcalico.org                   2021-10-02T15:34:35Z
imagesets.operator.tigera.io                          2021-10-02T15:34:36Z
ingressroutes.traefik.containo.us                     2021-10-02T16:51:37Z
ingressroutetcps.traefik.containo.us                  2021-10-02T16:51:37Z
ingressrouteudps.traefik.containo.us                  2021-10-02T16:51:37Z
installations.operator.tigera.io                      2021-10-02T15:34:36Z
installplans.operators.coreos.com                     2021-10-03T08:31:57Z
ipamblocks.crd.projectcalico.org                      2021-10-02T15:34:35Z
ipamconfigs.crd.projectcalico.org                     2021-10-02T15:34:35Z
ipamhandles.crd.projectcalico.org                     2021-10-02T15:34:35Z
ippools.crd.projectcalico.org                         2021-10-02T15:34:36Z
issuers.cert-manager.io                               2021-10-03T10:54:00Z
kubecontrollersconfigurations.crd.projectcalico.org   2021-10-02T15:34:36Z
middlewares.traefik.containo.us                       2021-10-02T16:51:38Z
middlewaretcps.traefik.containo.us                    2021-10-02T16:51:38Z
networkpolicies.crd.projectcalico.org                 2021-10-02T15:34:36Z
networksets.crd.projectcalico.org                     2021-10-02T15:34:36Z
operatorconditions.operators.coreos.com               2021-10-03T08:31:57Z
operatorgroups.operators.coreos.com                   2021-10-03T08:31:57Z
operators.operators.coreos.com                        2021-10-03T08:31:57Z
orders.acme.cert-manager.io                           2021-10-03T10:54:00Z
serverstransports.traefik.containo.us                 2021-10-02T16:51:38Z
subscriptions.operators.coreos.com                    2021-10-03T08:31:57Z
tigerastatuses.operator.tigera.io                     2021-10-02T15:34:36Z
tlsoptions.traefik.containo.us                        2021-10-02T16:51:38Z
tlsstores.traefik.containo.us                         2021-10-02T16:51:39Z
traefikservices.traefik.containo.us                   2021-10-02T16:51:39Z
```
![image](https://user-images.githubusercontent.com/458209/135752453-d995677e-d0d7-44a6-8905-c13e9c217d1b.png)

**Comment by eworld888** [comment]
My 1.23.1 cluster does not display CRD properly, but my 1.21 does

`Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.2", GitCommit:"8b5a19147530eaac9476b0ab82980b4088bbc1b2", GitTreeState:"clean", BuildDate:"2021-09-15T21:38:50Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.1", GitCommit:"86ec240af8cbd1b60bcc4c03c20da9b98005b92e", GitTreeState:"clean", BuildDate:"2021-12-16T11:34:54Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}`
![image](https://user-images.githubusercontent.com/4024822/147875656-d69a6c1a-0166-4ab9-b1cb-9df63e761b17.png)

**Comment by dkapanidis** [comment]
Solved on `v3.2.0`. This is a side-effect / duplicate of #309.

**Comment by pinkfloydx33** [comment]
I'm running 3.2.0 now and I do not get CRDs on 1.22.1; I do get it on 1.21.x and lower

**Comment by foal** [comment]
The same - issue is not fixed.

**Comment by dkapanidis** [comment]
fixed on `v3.4.0`.

(The issue was unrelated to #309)

CRDs were previously listed using API `apiextensions.k8s.io/v1beta1`, which is removed [since 1.22](https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/#api-changes).

Since Kubernetic v3.4.0 the CRD list is using the dynamic API client which doesn't depend on specific API versioning.

Tested with 1.22 and 1.17 for backwards compatibility.
